drf_spectacular/templates/drf_spectacular/swagger_ui.html

```diff
@@ -4,28 +4,27 @@
     <title>Swagger</title>
     <meta charset="utf-8"/>
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <link rel="stylesheet" type="text/css" href="//unpkg.com/swagger-ui-dist@3.35.0/swagger-ui.css" />
+    <link rel="icon" type="image/png" href="{{swagger_bundle_favicon_href}}"/>
+    <link rel="stylesheet" type="text/css" href="{{swagger_bundle_css_href}}" />
   </head>
   <body>
     <div id="swagger-ui"></div>
-    <script src="//unpkg.com/swagger-ui-dist@3.35.0/swagger-ui-bundle.js"></script>
+    <script src="{{swagger_bundle_js_href}}"></script>
     <script>
-    var swagger_settings  = {{ settings|safe }}
+    const swagger_settings  = {{ swagger_ui_settings|safe }}
     const ui = SwaggerUIBundle({
-        url: "{{ schema_url }}",
-        dom_id: '#swagger-ui',
-        presets: [
-          SwaggerUIBundle.presets.apis,
-          SwaggerUIBundle.SwaggerUIStandalonePreset
-        ],
-        layout: "BaseLayout",
-        deepLinking:true,
-        requestInterceptor: (request) => {
-          request.headers['X-CSRFToken'] = "{{ csrf_token }}"
-          return request;
-        },
-        ...swagger_settings
-      })
+      presets: [
```

Comment:
please restore the original version (16-27) as it was working perfectly fine. just add the `DownloadUrl`.

the ``...swagger_settings`` overrides existing entries. because of that  

```
    'SWAGGER_UI_SETTINGS': {
        'dom_id': '#swagger-ui',
        'layout': "BaseLayout",
        'deepLinking': True,
    },
```

is not necessary unless you wanted to change the defaults.

Reply:
restore  the original version (16-27) except `SwaggerUIBundle.SwaggerUIStandalonePreset`

<img width="603" alt="스크린샷 2020-10-12 오전 1 14 02" src="https://user-images.githubusercontent.com/24240623/95683762-4291ab00-0c28-11eb-8912-3cfb257d9c8a.png">